frontend/src/citizen-frontend/calendar/reservation-modal/form.spec.ts

```diff
@@ -1472,6 +1472,48 @@ describe('resetTimes', () => {
         ]
       })
     })
+    it('Closed holiday period + reservationLockedDueHolidayPeriod = false', () => {
+      const holidayPeriods = [
+        { period: selectedRange, state: 'closed' as const }
+      ]
+      const calendarDays: ReservationResponseDay[] = emptyCalendarDays.map(
+        (day) => ({
+          ...day,
+          children: [
+            { ...emptyChild, reservationLockedDueHolidayPeriod: false }
```

Comment:
The test would be more useful if you add another child that has `reservationLocked...: true`